Makefile

```diff
@@ -25,7 +25,15 @@ ${FISSILE_BINARY}: bin/dev/install_tools.sh
 ########## VAGRANT VM TARGETS ##########
 
 certs: uaa-certs
-	${GIT_ROOT}/bin/generate-dev-certs.sh cf bin/settings/certs.env
+	# If DOMAIN is set, place DOMAIN and UAA_HOST at the end of network.env temporarily
+	# Line continuations are necessary to trap EXIT after running script generation
+	if test -n "${DOMAIN}"; then \
+		echo "DOMAIN=${DOMAIN}" >> bin/settings/network.env; \
+		echo "UAA_HOST=uaa.${DOMAIN}" >> bin/settings/network.env; \
+		trap "sed -i '$d' bin/settings/network.env && \
+			sed -i '$d' bin/settings/network.env" EXIT; \
```

Comment:
This deletion does not seem to work, for me.
Also note the possible quoting hell (`''`-quoting inside of `""`-quoting.).

The outer `""`-quoting likely sees `$d` as variable to interpolate. Which is not defined. Thus the interpolation is empty. At that point the generated `sed` command looks like `sed -i '' bin/settings/network.env` and would do nothing, I believe. Which would match what I saw, with the appended lines not getting deleted.

`make cert -n` confirms.

I suspect the `$` will have to be `\`-escaped. No. That does not work either :(
It looks to be `make` itself doing the interpolation regardless of the quoting.
Adding a `\` generates `sed -i '\' bin/settings/network.env` which is not a usable sed command either.